lib/qa/authorities/linked_data/search_query.rb

```diff
@@ -37,7 +37,7 @@ def search(query, language: nil, replacements: {}, subauth: nil)
 
         def load_graph(url:, language:)
           @graph = Qa::LinkedData::GraphService.load_graph(url: url)
-          @graph = Qa::LinkedData::GraphService.filter(graph: @graph, language: language) unless language.blank?
+          @graph = Qa::LinkedData::GraphService.filter(graph: @graph, language: language, remove_blanknode_subjects: true)
```

Comment:
Don't need the `unless language.blank?` check because the GraphService will make that check.